internal/kafka/test/common/cluster_pollers.go

```diff
@@ -44,6 +44,7 @@ func WaitForClusterIDToBeAssigned(db *db.ConnectionFactory, clusterService *serv
 	return clusterID, NewPollerBuilder(db).
 		IntervalAndTimeout(defaultPollInterval, clusterIDAssignmentTimeout).
 		RetryLogMessagef("Waiting for an ID to be assigned to the cluster (%+v)", criteria).
+		DumpDB("clusters", "", "id", "created_at", "updated_at", "cluster_id", "external_id", "status").
```

Comment:
```suggestion
		DumpDB("clusters", "id", "created_at", "updated_at", "cluster_id", "external_id", "status").
```
trailing empty column?

Reply:
The second parameter of DumpDB is the filter. Empty string is correct if the whole table must be dumped.

Reply:
Thanks @ziccardi I should have checked the method signature correctly.